packages/places/src/useCases/dbManagement/dbWriter.js

```diff
@@ -0,0 +1,15 @@
+const Logger = require('../../utils/logger');
```

Comment:
Remeber to change the way we import logger

a) if you need to use Logger const { Logger } = require('@booking-services/shared');
b) if you need to reference logger file @booking-services/shared/src/utils/logger